keep-ui/app/alerts/alert-menu.tsx

```diff
@@ -161,7 +158,7 @@ export default function AlertMenu({
           leaveFrom="transform opacity-100 scale-100"
           leaveTo="transform opacity-0 scale-95"
         >
-          <Menu.Items className="z-50 relative mt-2 min-w-36 origin-top-right divide-y divide-gray-100 rounded-md bg-white shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none">
+          <Menu.Items className="z-50 fixed right-12 mt-2 divide-y divide-gray-100 rounded-md bg-white shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none">
```

Comment:
This makes the menu scroll down with the user scrolling. We need to think this through.

Reply:
Possibly resolved with https://github.com/keephq/keep/pull/657/commits/1fb3a84e30ff05642cb692a5f424627a34b75030. Lemme know what you think of the implementation. It's quite verbose, but the problem cannot (based on this [SO answer](https://stackoverflow.com/a/65535922/12627235)) addressed without dynamic positioning. 

Regarding the `@floatinui/react-dom` package, it's not installed as a dependency, but it's used in one of the other dependencies. I'm not sure – do we need to install it as a dependency?